internal/messagejournal/migrations/000001_messagejournal.down.sql

```diff
@@ -0,0 +1 @@
+DROP TABLE IF EXISTS journal;
```

Comment:
There is no empty line at the end of the file. Is it expected?

Reply:
I haven't seen any functional difference and my editor hasn't complained about it. I can't find any styling guidelines around this case either...maybe just leave it as it is and not add anything extra to migration scripts :shrug: